source/guides/ember-cli/installing.md

```diff
@@ -0,0 +1,62 @@
+## Got Node (and NPM)?
+
+If you're not sure if you have node.js installed, try running the following from your command line:
+
+```bash
+node --version
+```
+
+If the command works and you get back something like 0.10.x, you've already got it installed!
+
+If you **don't** have it installed...
+
+* Windows or Mac users [can simply download and run the installer](http://nodejs.org/download/).
+* Linux users [can check out this great guide](https://github.com/joyent/node/wiki/Installing-Node.js-via-package-manager) by Joyent for install instructions.
+
+Once you've got node.js installed, re-run the above ```node --version``` to verify your install.
+
+
+## Installing Ember
+
+Node Package Manager (npm) comes bundled with node.js and makes installing easy.
+
+Let's install Ember by running the following:
```

Comment:
This could be shortened to: 
> Installing Ember is done using [NPM](#)(link to installing NPM section). While you're at it we recommend you to also install [phantomjs](#) (if you don't have it already). Ember CLI uses phantomjs to run tests from the command line (without the need for a browser to be open).